src/main/java/task_at_7/CalculatorRunner.java

```diff
@@ -0,0 +1,90 @@
+package task_at_7;
+
+import java.io.IOException;
+import java.util.logging.*;
+
+
+public class CalculatorRunner {
+
+    private static final Logger logger = Logger.getLogger(CalculatorRunner.class.getName());
+
+    public static void main(String[] args) throws IOException {
+
+        LogManager.getLogManager().readConfiguration();
+        Handler fileHandler = new FileHandler("%h/myLog.log");
+        Handler consoleHandler = new ConsoleHandler();
+        logger.setUseParentHandlers(false);
+        logger.addHandler(fileHandler);
+        logger.addHandler(consoleHandler);
+        logger.config("Program starts right from here");
```

Comment:
this logger implementation should be separated to 'logger manager' class

Reply:
Got it.